server/services/licenceService.ts

```diff
@@ -149,6 +149,14 @@ export default class LicenceService {
     return this.licenceApiClient.createLicence(licence, user)
   }
 
+  private async getCRONumber(prisonerNumber: string, user: User): Promise<string> {
+    const prisoners = await this.prisonerService.searchPrisonersByNomisIds([prisonerNumber], user)
+    if (prisoners != null) {
```

Comment:
I don't think this will catch if `prisoners` is `undefined` rather than `null`.
I believe `if (prisoners)` should cover it.